lib/deprecation-helpers.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const { makeBadgeData } = require('./badge-data');
+
+const deprecatedServices = {
+  'gittip': new Date('2017-12-29'),
+  'gratipay': new Date('2017-12-29'),
+  'gemnasium': new Date('2018-05-15'),
+  'snap': new Date('2018-01-23'),
+  'snap-ci': new Date('2018-01-23'),
+  'cauditor': new Date('2018-02-15'),
+};
```

Comment:
Do you think it would make sense to put this in its own module, since it's data?